elastic/elastic-agent
# Fixing script to read ascii code

-- Bug

## What does this PR do?

WHAT: Updates makefile to read from different value of version.doc file
WHY: The `doc-branch` is a field that no longer is udpated. Thus we replace it with `stack-version` field

## Why is it important?

To avoid wrong references to main and alwasy reference to latest release of templates

## Checklist





## How to test this PR locally

1. Checkout PR
2. `cd elastic-agent/deploy/kubernetes`
3. Run make `make generate-k8s`

See in `elastic-agent-standalone-kubernetes.yaml`
```yaml
      #        wget -O - https://github.com/elastic/elastic-agent/archive/8.8.tar.gz | tar xz -C /etc/elastic-agent/inputs.d --strip=5 "elastic-agent-8.8/deploy/kubernetes/elastic-agent-standalone/templates.d"
```

- Relates https://github.com/elastic/elastic-agent/pull/3523
